packages/office-ui-fabric-react/src/utilities/keytips/KeytipManager.ts

```diff
@@ -13,8 +14,9 @@ export interface IUniqueKeytip {
 export class KeytipManager {
   private static _instance: KeytipManager = new KeytipManager();
 
-  public keytips: IUniqueKeytip[] = [];
-  public persistedKeytips: IUniqueKeytip[] = [];
+  public keytips: { [Key: string]: IUniqueKeytip } = {};
+  public persistedKeytips: { [Key: string]: IUniqueKeytip } = {};
```

Comment:
>{ [Key: string]: IUniqueKeytip }  [](start = 27, length = 33)

did you change them to maps to find them easier?

Reply:
Correct before whenever we wanted to find a specific keytips, we had to scan through all elements. By using a map, we get instant look-up.